src/taxonomy/tag-list/TagListTable.scss

```diff
@@ -0,0 +1,15 @@
+.tag-list-table {
+  .tag-table-expand-row {
+    float: right;
+  }
+
+  table tr:first-child > th:nth-child(2) > span {
+    // Used to move "Expand all" button to the right.
+    // Find the first <span> of the second <th> of the first <tr> of the <table>.
+    //
+    // The approach of the expand buttons cannot be applied here since the
+    // table headers are rendered differently and at the component level
+    // there is no control of this style.
+    float: right;
+  }
+}
```

Comment:
I think instead of this you should be able to use `d-flex justify-content-end` for the first one and use `display:flex; justify-content: flex-end` for the second. 

IMO using float can sometimes cause issues since it breaks the normal flow of the document.